CONTRIBUTING.md

```diff
@@ -123,6 +119,63 @@ use a shared working branch strategy.
 5. Resolve any merge conflicts.
 6. When your work is complete, merge the shared working branch into `develop` via a PR.
 
+### Working with docs-website-private
+
+The [docs-website-private repository](https://github.com/newrelic/docs-website-private) is a private copy of the docs-website repository. It is meant for private colloboration, on work that is not ready for public consumption.
+
+If you need to interact with the public & private repositories, there are three setups that you can use:
+1. cloning docs-website, adding a remote to the private repository.
+2. cloning docs-website-private, adding a remote to the public repository.
+3. doing both of the above.
+
+Our recommendation is setup #1, but you can use #2 or #3 to do the same things.
+
+#### General Workflow
+
+The general workflow is outlined [here](https://docs.google.com/document/d/1Lk16HGkWHlf52icq8AmaAiUp-lzFHEULCP7RAVjJ4aY/edit), and repeated below:
```

Comment:
This links to an internal document so I suggest we put whatever is in this doc into this contributing file if it's important enough to include.